#Goal
#bool call_mission
float32 x_goal
float32 y_goal

---
#Result
bool success

---
#Feedback
float32 x_status
float32 y_status